robottelo/ui/activationkey.py

```diff
@@ -85,6 +85,37 @@ def search_key(self, element_name):
             element = self.wait_until_element((strategy, value % element_name))
         return element
 
+    def search_key_subscriptions(self, ak_name, subscription_name):
+        """Fetch associated subscriptions from selected activation key"""
+
+        activation_key = self.search_key(ak_name)
+        if activation_key is None:
+            raise UINoSuchElementError(
+                'Could not find activation key {0}'.format(ak_name))
+        activation_key.click()
+        self.wait_for_ajax()
+
+        if self.wait_until_element(tab_locators["ak.subscriptions"]) is None:
+            raise UINoSuchElementError("Couldn't find Subscriptions tab")
+        self.find_element(tab_locators["ak.subscriptions"]).click()
+        self.wait_for_ajax()
+
+        searchbox = self.wait_until_element(
+            locators["ak.subscriptions.search"])
+        if searchbox is None:
+            raise UINoSuchElementError(
+                "Couldn't find Subscriptions search box")
+        searchbox.clear()
+        searchbox.send_keys(escape_search(subscription_name))
+        self.find_element(locators["ak.subscriptions.search_button"]).click()
+        self.wait_for_ajax()
+
+        strategy, value = locators["ak.get_subscription_name"]
+        element = self.wait_until_element(
```

Comment:
Unfortunately, I can't - mentioned function is designed to work only with searchboxes which are matching locator ```common_locators["search"]``` or ```common_locators["kt_search"]```. Activation key subscription searchbox locator doesn't match any of these.
However I can try to update ```search_entity()``` to receive optional ```searchbox_locator``` and ```seach_button_locator``` arguments, if that's worth it (and I'm not sure whether it is, as ```search_entity()``` is commonly-used function so I can potentially create a huge impact. In addition, there's no guarantee those new arguments will be used somewhere else except this one case). @elyezer , @jyejare, your thoughts on this?